### PR TITLE
Removed cyclic dependency to mwe2.launch

### PR DESCRIPTION
**Pull request by szarnekow**

### CODE REVIEW

**Comment by szarnekow** [comment]
FYI @cdietrich

File: org.eclipse.xtend.core/build.properties [hunk 1]
```diff
@@ -13,6 +13,5 @@ exclude.. = **/GenerateXtend.mwe2
 src.includes = about.html
 src.excludes = generator/,\
                generator-xtend-gen/
-additional.bundles = org.eclipse.emf.mwe2.launch,\
```
Comment (cdietrich) [inline org.eclipse.xtend.core/build.properties:16]: did you try if run workflow works? for me it does not.
grafik

Comment (szarnekow) [reply org.eclipse.xtend.core/build.properties:16]: Of course, my bad
We need to port the mwe2 file (back) to Java

Comment (cdietrich) [reply org.eclipse.xtend.core/build.properties:16]: or we keep launch here

Comment (szarnekow) [reply org.eclipse.xtend.core/build.properties]: Yes, I just pushed that. I misread the error message. Multitasking is hard.

**Review by cdietrich — APPROVED** [review]

**Review by LorenzoBettini — APPROVED** [review]

**Pull request merged**